analytics/policy/docs/policy_evaluation_architecture.drawio

```diff
@@ -0,0 +1,259 @@
+<mxfile host="Electron" modified="2024-01-27T07:52:55.513Z" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/22.1.21 Chrome/120.0.6099.109 Electron/28.1.0 Safari/537.36" etag="y_NmPirg3YK9W47xU4dC" version="22.1.21" type="device">
+  <diagram name="Page-1" id="Qe4asFTC23VRKxSRaJ4f">
+    <mxGraphModel dx="3434" dy="2751" grid="1" gridSize="18" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0">
```

Comment:
same thing here. try to have a single source of truth.

Reply:
> same thing here. try to have a single source of truth.

this one is the drawio file though, i dont have it in other places

Reply:
can this drawio be added to architecture page instead?

Reply:
> can this drawio be added to architecture page instead?

I can add a link to the wiki page, but then would need to maintain in 2 places.
I will remove this for now